packages/manager/apps/pci-private-network/src/pages/listing/ListingLayout.component.tsx

```diff
@@ -0,0 +1,112 @@
+import { useCallback, useEffect } from 'react';
+import { Outlet, useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import {
+  PciGuidesHeader,
+  useProjectUrl,
+  Title,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import {
+  OsdsBreadcrumb,
+  OsdsSpinner,
+  OsdsTabBar,
+  OsdsTabBarItem,
+  OsdsTabs,
+} from '@ovhcloud/ods-components/react';
+import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import {
+  PciAnnouncementBanner,
+  PciDiscoveryBanner,
+  useProject,
+} from '@ovh-ux/manager-pci-common';
+import { usePrivateNetworks } from '@/data/hooks/networks/useNetworks';
+import { PrivateNetworkTabName } from './ListingLayout.constant';
+import { useActiveTab } from '@/hooks/useActiveTab/useActiveTab';
+
+const ListingLayout: React.FC = () => {
+  const { t } = useTranslation('listing');
+  const projectUrl = useProjectUrl('public-cloud');
+  const { data: project } = useProject();
+  const projectId = project.project_id;
+  const navigate = useNavigate();
+  const tab = useActiveTab();
+
+  const { isPending, data: networks } = usePrivateNetworks(projectId);
+
+  const { clearNotifications } = useNotifications();
+
+  const handleTabChange = useCallback(({ detail }: CustomEvent) => {
+    clearNotifications();
+
+    if (detail?.panel === PrivateNetworkTabName.LOCAL_ZONE_TAB_NAME) {
+      navigate('./localZone');
+    } else {
+      navigate('');
```

Comment:
really necessary ?

Reply:
Yes it's event handler on tabsChanged because we are managing tabs as two different pages with their own route. Otherwise, it will never be redirect